templates/compute-domain-daemon.tmpl.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         {{ .ComputeDomainLabelKey }}: {{ .ComputeDomainLabelValue }}
       containers:
       - name: compute-domain-daemon
-        image: ubuntu:22.04
+        image: nvcr.io/nvidia/cuda:12.2.0-base-ubuntu22.04
```

Comment:
Please use the latest image: `nvcr.io/nvidia/cuda:12.8.1-base-ubuntu22.04`

Out of scope: we may want to templatize the image path here and find a way to keep it up-to-date.

Reply:
> we may want to templatize the image path here

Absolutely. We have to allow for orgs to take control of their supply chain. I also just mentioned this in https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/315#issuecomment-2796775389.